docs/hooks-go.md

```diff
@@ -10,21 +10,49 @@ Go hooks are using [Dredd's hooks handler socket interface](hooks-new-language.m
 
 ```
 $ go get github.com/snikch/goodman
+$ cd $GOPATH/src/github.com/snikch/goodman
+$ go build -o $GOPATH/bin/goodman github.com/snikch/goodman/bin
 ```
 
 ## Usage
 
-Using Dredd with Go is slightly different to other languages, as a binary needs to be compiled for execution. Instead of providing a variety of hookfiles to run, your own binary is built using the Go hooks library `goodman` to run your specific hooks. The path to this binary is passed to dredd via the `--language` flag. You may also need to supply a `--hookfiles` flag to get the tests to run, but the value of this is of no matter (the dredd runner requires it).
+Using Dredd with Go is slightly different to other languages, as a binary needs to be compiled for execution. The --hookfiles flags will be compiled hook binaries.  See below for an example hooks.go file to get an idea of what this would look like.
 
 ```
-$ dredd apiary.apib http://localhost:3000 --language ./my-dredd-hook-server --hookfiles *.go
+$ dredd apiary.apib http://localhost:3000 --server ./go-lang-web-server-to-test --language go --hookfiles ./hook-file-binary
```

Comment:
@pksunkara the --server option is the command to get the server that will be under test started.  So the command assumes that the `./go-lang-web-server` is a go web service that will start a webserver that listens on localhost port 3000.